### ISSUE TITLE
Links in comments open on touch instead of release.

### ISSUE THREAD

**Issue by VolkerA4** [futo-org/grayjay-android#875]
When touching links posted in comments they open immediately, before the user has released the selection. This results in accidental selections (particularly when scrolling), and differs from how links are handled inside descriptions. Inside descriptions, links are opened when the user releases their selection.

This, open on release functionality, is the more desirable solution. Resulting in fewer accidental selections. And aligning with how links are handled elsewhere in Grayjay and other applications (IE, YouTube).

**Comment by Doubs115** [comment]
Yes incredibly frusterating, please hotfix.

**Comment by nastynick64** [comment]
Agreed! A lot of YouTubers like to put links in the comments, I have accidentally opened links many time when I just wanted to see the comments. Although, it might be even more annoying when I accidentally scroll over a comment with a timestamp on it, cause that causes it to jump to that timestamp in the video making it difficult to go back to where I was in the video (especially if the video is a long one)
